ocean_provider/config.py

```diff
@@ -56,6 +57,11 @@
         "resources",
     ],
     NAME_STORAGE_PATH: ["STORAGE_PATH", "Path to the local database file", "resources"],
+    NAME_BLOCK_CONFIRMATIONS: [
+        "BLOCK_CONFIRMATIONS",
```

Comment:
I'm adding this as just a comment (instead of a change request) because this problem should go away once we remove the ocean.py dependency from the provider in https://github.com/oceanprotocol/provider/issues/218

Allowing provider `Config` and ocean.py `Config` to use the same `BLOCK_CONFIRMATIONS` envvar and `block_confirmations` config option creates unexpected coupling between the two code bases. It would be better to add a prefix to differentiate the two. `PROVIDER_BLOCK_CONFIRMATIONS` & `provider_block_confirmations`

Reply:
As I understand it, this is a temporary fix, where Alex bumped the ocean.py because he doesn't have time to review ocean.py removal right now.